src/Bridges/Constraint/bridges/square.jl

```diff
@@ -104,31 +112,20 @@ function bridge_constraint(
             k += 1
             push!(upper_triangle_indices, k)
             # We constrain the entries (i, j) and (j, i) to be equal
-            upper = f_scalars[i+(j-1)*dim]
-            lower = f_scalars[j+(i-1)*dim]
-            diff = MOIU.operate!(-, T, upper, lower)
-            MOIU.canonicalize!(diff)
+            f_ij, f_ji = f_scalars[i+(j-1)*dim], f_scalars[j+(i-1)*dim]
+            diff = MOI.Utilities.operate!(-, T, f_ij, f_ji)
+            MOI.Utilities.canonicalize!(diff)
             # The value 1e-10 was decided in https://github.com/jump-dev/JuMP.jl/pull/976
             # This avoid generating symmetrization constraints when the
             # functions at entries (i, j) and (j, i) are almost identical
-            if !MOIU.isapprox_zero(diff, 1e-10)
-                if MOIU.isapprox_zero(diff, 1e-8)
-                    @warn "The entries ($i, $j) and ($j, $i) of the" *
```

Comment:
Why have you removed this ?

Reply:
I removed this for a number of reasons:
 * The warning wasn't tested
 * Why even have the warning? The user wrote something. They expect it to work. If it's a problem, they can explicitly form a symmetric matrix.
 * Is there any code in the wild that actually triggers this warning?
 * Why 1-e8 and 1e-10? That's a pretty narrow band. It's unlikely to trigger from any linear algebra calculations, since that's going to result in errors much smaller than 1e-10. But if it comes from the solution of another optimizer, the error is likely 1e-6, which won't trigger this warning.